lib/active_merchant/billing/gateways/mercury.rb

```diff
@@ -103,7 +103,7 @@ def build_non_authorized_request(action, money, credit_card, options)
           xml.tag! 'Transaction' do
             xml.tag! 'TranType', 'Credit'
             xml.tag! 'TranCode', action
-            if options[:allow_partial_auth] && (action == 'PreAuth' || action == 'Sale')
+            if options[:allow_partial_auth] && ['PreAuth', 'Sale'].include?(action)
```

Comment:
I don't know if this was intended to go in with these changes, but it looks fine.

Reply:
It was; RuboCop 0.58.1 has a new check, and that was the only violator, so I just figured I'd roll with it as part of the RuboCop upgrade.